### PR TITLE
feat(core.js): allow for turning on and off the masking

### PR DESCRIPTION
**Pull request by Umi8Umi**
### Description
Add a prop to use the maskedValue even when the mask is an empty string and maintains the cursor position

#### Coverage
```bash
---------------|----------|----------|----------|----------|-------------------|
File           |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
---------------|----------|----------|----------|----------|-------------------|
All files      |    99.38 |    94.66 |    96.88 |      100 |                   |
 component.vue |      100 |    91.67 |      100 |      100 |               103 |
 core.js       |      100 |    96.97 |     87.5 |      100 |           112,142 |
 directive.js  |      100 |      100 |      100 |      100 |                   |
 masker.js     |    98.39 |    91.49 |      100 |      100 |         12,63,105 |
 tokens.js     |      100 |      100 |      100 |      100 |                   |
---------------|----------|----------|----------|----------|-------------------|

Test Suites: 5 passed, 5 total
Tests:       62 passed, 62 total
Snapshots:   0 total
Time:        2.545s, estimated 3s
```

### CODE REVIEW

**Review by Robo-Rin — APPROVED** [review]

**Review by RonaldJerez — COMMENTED** [review]
Thanks Sarah, these look good.  Just waiting on confirmation from our api services that passing the masked value indeed works as expected before we merge this, as otherwise we may not need it.

**Pull request merged**
